tools/release/java/src/main/java/com/ibm/icu/dev/tools/docs/StableAPI.java

```diff
@@ -218,7 +218,8 @@ private InputStream loadStream(String name, String argName, File argFile) {
     	InputStream stream = null;
     	if(argFile != null) {
     		try {
-	    		stream = new FileInputStream(argFile);
+                stream = new FileInputStream(argFile);
```

Comment:
funky indentation here & around... but i will approve anyway in the interest of the rc

Reply:
Will re-indent in a new PR

Reply:
reindented in #933